conf/services/pay-what-you-owe-in-instalments.yml

```diff
@@ -7,29 +7,29 @@ serviceUrl: /pay-what-you-owe-in-instalments
 contactFrontendServiceId: pay-frontend
 complianceStatus: partial
 accessibilityProblems:
-  - some error messages are not descriptive enough for people with visual impairments - but are compatible with common assistive technology tools
-  - some headings are not consistent
-  - elements which are missing their labels - using a screen reader to read back the entire page will provide all the information needed
+  - On the 'Which day do you want to pay each month?' and 'How much can you pay each month?' pages, the label is missing for the text inputs making it difficult to know what should be entered.
```

Comment:
these look like they may be the wrong type of quotes - are these straight quotes rather than curly quotes that I can see in the welsh version of this file?

Reply:
corrected